### DISCUSSION TITLE
HM800 - Keine Verbindung

### DISCUSSION THREAD

**Discussion by ghost** [tbnobody/OpenDTU#1452] [Q&A]
Ich bekomme keine Verbindungf zu meinem HM800. Ich vermute diese Meldung:

E (632) esp_core_dump_flash: No core dupartition found!
E (632) esp_core_dump_flash: No core dump partition found!

als Ursache. Könnte mir jemand helfen? ( inverter S/N habe ich anonymisiert)

![image](https://github.com/tbnobody/OpenDTU/assets/123190453/956572ab-9731-4370-9adb-a72d31a623e7)

Log:

rst:0xc (SW_CPU_RESET),boot:0x13 (SPI_FAST_FLASH_BOOT)
configsip: 0, SPIWP:0xee
clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00
mode:DIO, clock div:2
load:0x3fff0030,len:1184
load:0x40078000,len:13192
load:0x40080400,len:3028
entry 0x400805e4
E (632) esp_core_dump_flash: No core dupartition found!
E (632) esp_core_dump_flash: No core dump partition found!

Starting OpenDTU
Initialize FS... done
Reading configuration... done
Reading PinMapping... found valid mapping done
Initialize Network... done
Setting Hostname... Configuring WiFi STA using new credentials... done
Initialize NTP... done
Initialize SunPosition... done
Initialize MqTT... done
Initialize WebApi... done
Initialize Display... done
Initialize LEDs... done
Check for default DTU serial... done
Initialize Hoymiles interface... NRF: Connection successful
  Setting radio PA level... 
  Setting DTU serial... 
  Setting poll interval... 
  Adding inverter: 11111111111 - HM800 done
done
Switch to WiFi mode
Setting Hostname... done
Configuring WiFi STA using new credentials... done
Configuring WiFi STA static IP... done
WiFi connected
WiFi got ip: 192.168.1.170
Network connected
Fetch inverter: 1111111111
TX RealTimeRunData Channel: 23 --> 15 90 52 43 04 80 12 36 92 80 0B 00 65 3A 8C 17 00 00 00 00 00 00 00 00 B9 DF 8F 
RX Period End
All missing
Nothing received, resend whole request
TX RealTimeRunData Channel: 40 --> 15 90 52 43 04 80 12 36 92 80 0B 00 65 3A 8C 17 00 00 00 00 00 00 00 00 B9 DF 8F 
RX Period End
All missing
Nothing received, resend whole request
TX RealTimeRunData Channel: 61 --> 15 90 52 43 04 80 12 36 92 80 0B 00 65 3A 8C 17 00 00 00 00 00 00 00 00 B9 DF 8F 
Admin AP remaining seconds: 10 / 180
RX Period End
All missing
Nothing received, resend whole request
TX RealTimeRunData Channel: 75 --> 15 90 52 43 04 80 12 36 92 80 0B 00 65 3A 8C 17 00 00 00 00 00 00 00 00 B9 DF 8F 
RX Period End
All missing
Nothing received, resend whole request
TX RealTimeRunData Channel: 3 --> 15 90 52 43 04 80 12 36 92 80 0B 00 65 3A 8C 17 00 00 00 00 00 00 00 00 B9 DF 8F 
RX Period End
All missing

**Comment by stefan123t** [comment]
Das hört sich an als hattest Du nur das Firmware Binary ohne die Partition mit den Konfig Dateien geflasht. Nach welcher Anleitung hast Du geflasht und vermutlich musst Du es nochmal machen. Viel Erfolg!

**Reply by ghost** [reply]
Ich es mit [Flash with ESP_Flasher](https://github.com/tbnobody/OpenDTU#flash-with-esp_flasher-windows) geflasht. Den Hinweis im Text bzgl. der älteren Versionen auf der HP habe ich überlesen und einfach die aktuelle Version in den Beschriebenen Speicher Bereich geschrieben. Ich vermute das beim erstmaligen flashen auf die aktuelle Version etwas schief gelaufen ist. Ich habe jetzt schon duzende Male verschiedene Versionen mit verschiedenen Tools genutzt um den Fehler zu beheben. Leider ohne Erfolg. Wo bekomme ich den die Konfig Daten her die man eigentlich vor dem flashen auf eine aktuelle Version vor 15.03.2023 benötigt. Ich dachte diese wären in den neuen Versionen bereits Bestandteil?!

**Reply by stefan123t** [reply]
Hallo @ch-r-is-7
Probiere es doch mal mit dem WebFlasher (nur Chromium, kein Firefox) da werden auch die Partitionen geflasht. Mit dem ESP Flasher kenne ich mich nicht aus. Sonst melde Dich im Discord vielleicht kann man Dir da besser helfen ?
